Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.redhat.io/ubi8:latest as builder
```

Comment:
My last suggestion would be to pin this to a versioned tag rather than `:latest` - like we do in approval-api and others.

https://github.com/lindgrenj6/approval-api/blob/297c4a2e3e57feea651ed6447c0b4f34ff07a160/Dockerfile#L1